Add Azure to Cloud Provider References

kwcanuck opened the pull request:
Resolves #1012 

@dnarain-astro when time permits, could you validate the new Azure section that was added to _Create a new cluster on Astro_ (create-cluster.md). Thank you!

dnarain-astro commented on astro/create-cluster.md:
```diff
@@ -28,11 +28,12 @@ To create an Astro cluster, you need to have:
 <TabItem value="aws">
 
-#### Step 1: Submit a request to Astronomer
+#### Step 1: Submit a request to Astronomer support
```
Since the doc only updated Azure and is 100% accurate, we should publish.

But, for a future iteration (maybe a new issue?): For all 3 clouds - we don't ask for VPC/VNet ranges and account for any of them.  We should ask for both of these.  We can default to "whatever you used for your existing cluster"

If they have 2 clusters, then I have no clue what we will say we default to - their last created cluster? their first cluster? no defaults they must specify?

kwcanuck replied:
@paolaperaza or @jwitz who might be able to determine the default values for VPC/VNet ranges?

dnarain-astro commented:
Here are the defaults we have for VPCs:
* AWS

   - A CIDR block (RFC 1918 IP Space) no smaller than a /19 range. You must
   ensure it does not overlap with the AWS VPC(s) that you will be peering
   with later. The default CIDR range is 172.20.0.0/19.

* GCP

   - *Subnet CIDR*: Used by nodes in your GKE cluster (Default:
   172.20.0.0/19)



* Azure


   - Optional. Your custom CIDR ranges for Astronomer service connections.
   The default is 172.20.0.0/19.


On Wed, Aug 10, 2022 at 10:43 AM Rob Gray ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In astro/create-cluster.md
> <https://github.com/astronomer/docs/pull/1020#discussion_r942731469>:
>
> >          {label: 'GCP', value: 'gcp'},
>      ]}>
>  <TabItem value="aws">
>
> -#### Step 1: Submit a request to Astronomer
> +#### Step 1: Submit a request to Astronomer support
>
> @paolaperaza <https://github.com/paolaperaza> or @jwitz
> <https://github.com/jwitz> who might be able to determine the default
> values for VPC/VNet ranges?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/astronomer/docs/pull/1020#discussion_r942731469>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWLQX2CIHCYSGFFPRFBQYQ3VYPS5RANCNFSM56EUONCQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

kwcanuck commented:
Hi, @dnarain-astro.

As discussed yesterday, I have added information about the default VPC settings for AWS, Azure, and GPC. This information appears in the _Create a new cluster on Astro_ topic. I have also added a new section titled _VPC peering prerequisites (Optional)_ to the topic _Install Astro on Azure_.  You will need to validate this text for technical accuracy.

Thanks again for your help to date.

dnarain-astro commented on astro/create-cluster.md:
```diff
@@ -42,7 +43,7 @@ To create a new Astro cluster on AWS for your Organization, submit a [support re
 - Your preferred node instance type
 - Your preferred max node count
```
```suggestion
- Your preferred max node count
- Your preferred VPC CIDR
- Your preferred region
```

dnarain-astro commented on astro/create-cluster.md:
```diff
@@ -42,7 +43,7 @@ To create a new Astro cluster on AWS for your Organization, submit a [support re
 - Your preferred max node count
 
-If you don't specify configuration preferences, Astronomer creates a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [AWS resource reference](resource-reference-aws.md).
+If you don't specify configuration preferences, Astronomer support creates a cluster with two `m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. If you're using Virtual Private Cloud (VPC) peering, a CIDR block (RFC 1918 IP Space) with the default CIDR range `172.20.0.0/19` is implemented. For information on all supported regions, configurations, and defaults, see [Resources require ...
```
```suggestion
If you don't specify configuration preferences, Astronomer support creates a cluster with a VPC CIDR of 172.20.0.0/19,`m5.xlarge` nodes and a maximum node count of 20 in `us-east-1`. For information on all supported regions, configurations, and defaults, see [Resources required for Astro on AWS](resource-reference-aws.md).
```

dnarain-astro commented on astro/create-cluster.md:
```diff
@@ -104,31 +105,50 @@ To create a cluster in one of these regions, complete the following additional s
 - The GCP region that you want to host your cluster in.
 - Your preferred node instance type.
 - Your preferred CloudSQL instance type.
 - Your preferred maximum node count.
```
- Your preferred VPC CIDR

dnarain-astro commented on astro/create-cluster.md:
```diff
@@ -104,31 +105,50 @@ To create a cluster in one of these regions, complete the following additional s
 - Your preferred maximum node count.
 
-If not specified, Astronomer will create a cluster with `e2-medium-4 nodes`, one Medium General Purpose CloudSQL instance (4vCPU, 16GB), and a maximum node count of 20 in `us-central1`.
+If you don't specify configuration preferences, Astronomer support creates a cluster with `e2-medium-4 nodes`, one Medium General Purpose CloudSQL instance (4vCPU, 16GB), and a maximum node count of 20 in `us-central1`. If you're using Virtual Private Cloud (VPC) peering, the default Subnet CIDR `172.20.0.0/19` is implemented. For information on all supported regions and configurations, see [Reso ...
```
```suggestion
If you don't specify configuration preferences, Astronomer support creates a cluster with a VPC CIDR of 172.20.0.0/19,  `e2-medium-4 nodes`, one Medium General Purpose CloudSQL instance (4vCPU, 16GB), and a maximum node count of 20 in `us-central1`.  For information on all supported regions and configurations, see [Resources required for Astro on GCP](resource-reference-gcp.md).  
```

jwitz reviewed (approved):
No comments 🏆 LGTM!
